lib/meals/requests.js

```diff
@@ -25,14 +25,19 @@ export const addFoodToMeal = (food, meal) => {
   let calories = food.getElementsByTagName('p')[1].innerText
   let data = { name: name, calories: calories }
```

Comment:
Why do you need this if we also have the object below?

Reply:
Oh great catch. Let me check into this!